src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ResponseHandlers.scala

```diff
@@ -198,6 +198,7 @@ object ResponseHandlers extends AkkaResponses {
     result match {
       case Success(s)                              => success(s)
       case Failure(ex: OperationForbidden.type)    => forbidden(ex, logMessage)
+      case Failure(ex: InterfaceAlreadyExists)     => forbidden(ex, logMessage)
```

Comment:
not related to authorization itself
```suggestion
      case Failure(ex: InterfaceAlreadyExists)     => badRequest(ex, logMessage)
```